charts/ingress-nginx/templates/controller-service.yaml

```diff
@@ -28,6 +28,9 @@ spec:
 {{- if .Values.controller.service.loadBalancerSourceRanges }}
   loadBalancerSourceRanges: {{ toYaml .Values.controller.service.loadBalancerSourceRanges | nindent 4 }}
 {{- end }}
+{{- if .Values.controller.service.loadBalancerClass }}
+  loadBalancerClass: {{ toYaml .Values.controller.service.loadBalancerClass }}
```

Comment:
The value `controller.service.loadBalancerClass` is a string. Why `toYaml` is used here?